### PR TITLE
More changes to indicate Python 3 requirement

### PR DESCRIPTION
**Pull request by takluyver**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -239,6 +239,7 @@ def run(self):
 extras_require['all'] = everything
 
 if 'setuptools' in sys.modules:
+    setuptools_extra_args['python_requires'] = '>=3.3'
```
Comment (takluyver) [inline setup.py]: This requires setuptools >= 24.2.0:
https://setuptools.readthedocs.io/en/latest/history.html#v24-2-0

Comment (damianavila) [inline setup.py]: Don't you need to require this on TravisCI? https://github.com/ipython/ipython/blob/master/.travis.yml#L13

Btw, should not be something to detect/request this requirement at the setup.py level?

**Comment by Carreau** [comment]
🍰  !

I think this would likely need an announce on the mailing list.

File: setup.py (hunk 1 above)
Comment (damianavila) [inline setup.py]: @takluyver ^^^, in case you miss the previous notification :wink:

Comment (takluyver) [inline setup.py]: Thanks, yes, we should bump the requirement on Travis.

**Pull request merged**